library/core/src/ptr/alignment.rs

```diff
@@ -240,6 +240,7 @@ type AlignmentEnum = AlignmentEnum32;
 #[cfg(target_pointer_width = "64")]
 type AlignmentEnum = AlignmentEnum64;
 
+#[cfg(target_pointer_width = "16")]
 #[derive(Copy, Clone, PartialEq, Eq)]
 #[repr(u16)]
 enum AlignmentEnum16 {
```

Comment:
If these are *only* used for the `type AlignmentEnum` alias above, I wonder why we shouldn't just define them as `enum AlignmentEnum` directly.